Gordon360/Controllers/HousingController.cs

```diff
@@ -207,18 +207,17 @@ public ActionResult<ApartmentApplicationViewModel> GetApartmentApplication(int a
             //get token data from context, username is the username of current logged in person
             var authenticatedUserUsername = AuthUtils.GetUsername(User);
 
-            string userID = _accountService.GetAccountByUsername(authenticatedUserUsername).GordonID;
+            string username = _accountService.GetAccountByUsername(authenticatedUserUsername).ADUserName;
```

Comment:
Hmmm - something doesn't seem right here...

Reply:
It took me a moment...